elixir/README.md

```diff
@@ -0,0 +1,43 @@
+# Elixir
+
+This document covers some of the important concepts for Elixir that a developer must learn. These may cover some very basic concepts to advanced concepts as well.
+
+## Official documentation
+
+1. [Elixir](https://elixir-lang.org/docs.html)
+
+## Online resources
+
+#### Functional programming
+
+1. [Functional Programming in 40 Minutes](https://www.youtube.com/watch?v=0if71HOyVjY)
+
+#### Learn Elixir
+
+1. [Elixircasts](https://elixircasts.io/series/learn-elixir)
+1. [Dwyl](https://github.com/dwyl/learn-elixir)
+
+#### Learn Elixir and Phoenix
+
+1. [Alchemist Camp](https://alchemist.camp/)
+
+#### Elixir Coarses
```

Comment:
@AkhileshNegi This branch is already merged, but we need to correct this typo. Could you please fix this?
cc @pankaj-ag